### PR TITLE
[_]:(chore) New services structure based on products

### PR DESCRIPTION
**Pull request by PixoDev**
This PR provides a first iteration of a new folder structure for mobile services

**Motivation:**

We need some modularization in mobile, since it contains basically more than one Internxt products.

**Solution:**

Provide a new service folder organization based on products.

services
-- drive
-- photos
-- common

Each service file is contained in a folder, the folder ideally should have this structure in the future:

drive
-- file
---- driveFile.service.ts
---- driveFile.spec.ts
---- index.ts

Benefits:

- Modularized code, scoped by products
- Each file has their own test file, which ideally serve as documentation of the purpose of that file
- We can remove/add an entire product easily using this structure

### CODE REVIEW

**Review by masterprog-cmd — APPROVED** [review]

**Comment by sg-gs** [comment]
I have to thank you for the comment on the PR, as is a very descriptive one which could be useful in case we have to backtrack any issue.

File: src/services/drive/folder/driveFolder.service.ts [hunk 1]
```diff
@@ -68,7 +68,7 @@ class DriveFolderService {
         const fileFullName = `${file.name}${file.type ? '.' + file.type : ''}`;
         const relativePath = `${currentFolder.relativePath}/${fileFullName}`;
 
-        fileService.renameFileInNetwork(file.fileId, bucketId, relativePath);
+        driveFileService.renameFileInNetwork(file.fileId, bucketId, relativePath);
```
Comment (sg-gs) [inline src/services/drive/folder/driveFolder.service.ts:71]: shouldn't this be - according to what has been done here - ? `drive.file.rename...`

File: src/services/drive/folder/driveFolder.service.ts [hunk 2]
```diff
@@ -50,7 +50,7 @@ class DriveFolderService {
 
     while (pendingFolders.length > 0) {
       const currentFolder = pendingFolders[0];
-      const folderContentResponse = await fileService.getFolderContent(currentFolder.folderId);
+      const folderContentResponse = await driveFileService.getFolderContent(currentFolder.folderId);
```
Comment (sg-gs) [inline src/services/drive/folder/driveFolder.service.ts:53]: here too: `drive.file.getFolderContent` ?

**Review by sg-gs — CHANGES_REQUESTED** [review]

File: src/services/drive/folder/driveFolder.service.ts (hunk 2 above)
Comment (PixoDev) [reply src/services/drive/folder/driveFolder.service.ts:53]: Missed this ones, since they are relative, updating

**Review by sg-gs — APPROVED** [review]

**Pull request merged**